packages/odyssey-storybook/src/components/odyssey-mui/Tabs/Tabs.stories.tsx

```diff
@@ -168,3 +185,52 @@ export const Icons: StoryObj<TabItemProps> = {
     selectTab({ canvasElement, step })("Tab Icon", "Xenomorphs");
   },
 };
+
+export const Controlled: StoryObj<TabItemProps> = {
+  render: function C() {
+    const [value, setValue] = useState("planets");
+
+    const onChange = (_e: unknown, value: string) => {
+      setValue(value);
+    };
+
+    const tabs: TabItemProps[] = [
+      {
+        label: "Planets",
+        value: "planets",
+        children: "Information about Planets",
+      },
+      {
+        label: "Moons",
+        value: "moons",
+        children: "Information about Moons",
+      },
+      {
+        label: "Galaxies",
+        value: "galaxies",
+        children: "Information about Galaxies",
+      },
+    ];
+
+    return (
+      <>
+        <Tabs
+          value={value}
+          ariaLabel="controlled tabs example"
+          tabs={tabs}
+          onChange={onChange}
+        />
+        <br />
```

Comment:
We shouldn't be using `<br />` anywhere in Odyssey, even in our stories. Someone _will_ copy paste that :/.

Check with @jordankoschei-okta on the UI for this story:
![image](https://github.com/okta/odyssey/assets/97464104/90467a52-4dd1-454d-9b16-3698d6305b22)
He'll have more input on how it should look.

https://aff2083.ods.dev/?path=/story/mui-components-tabs--controlled

Reply:
I approved this, but I recommend removing the "Current tab is galaxies" text and changing the button to read "Navigate to Galaxies tab". The "current tab..." text isn't needed